multicluster/apis/multicluster/v1alpha1/resourceimport_types.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+	mcs "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ResourceImportSpec defines the desired state of ResourceImport
+type ResourceImportSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// ClusterID specifies the member cluster this resource to import to.
+	// When not specified, import to all member clusters.
+	ClusterID string `json:"clusterID,omitempty"`
```

Comment:
if user or admin can specify which member cluster this ResourceImport can be imported to, should it be a slice to allow multiple clusters instead of one or all? and one more question is who and when this kind of info will be updated? I didn't see any fields can specify such info in ResourceExport.

Reply:
good point. Will make ClusterIDs a slice,

Reply:
done.